SeoulBikeWithPollen/latlon_to_dong.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import json
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from bs4 import BeautifulSoup
+from selenium.webdriver.common.by import By
+import time
+from shapely.geometry import Point, Polygon, LineString
+import geopandas as gpd
+
+# gdf_admin_gu_pg = gpd.read_file('SeoulBikeWithPollen\dong_pg\BND_ADM_DONG_PG.shp', encoding='cp949')  
+# print(gdf_admin_gu_pg.head())
+
+key = open("key.txt", 'r').read()
+url1 = f'http://openapi.seoul.go.kr:8088/{key}/json/bikeList/1/1000/'
+url2 = f'http://openapi.seoul.go.kr:8088/{key}/json/bikeList/1001/2000/'
+url3 = f'http://openapi.seoul.go.kr:8088/{key}/json/bikeList/2001/2694/'
+
+response1 = requests.get(url1).json()
+response2 = requests.get(url2).json()
+response3 = requests.get(url3).json()
+res1 = response1["rentBikeStatus"]["row"]
+res2 = response2["rentBikeStatus"]["row"]
+res3 = response3["rentBikeStatus"]["row"]
+ress = res1 + res2 + res3
+print(len(ress))
```

Comment:
```suggestion
def get_bike_data(start, end):
    key = open("key.txt", 'r').read()
    url = f'http://openapi.seoul.go.kr:8088/{key}/json/bikeList/{start}/{end}/'
    response = requests.get(url).json()
    return response["rentBikeStatus"]["row"]

total_bike_count = 0
for start in range(1, 2700, 1000):
    end = min(start + 999, 2694)
    response = get_bike_data(start, end)
    total_bike_count += len(response)

print(total_bike_count)
```

가독성/재사용성 고려하여 코드를 간결하게 정리해보면 좋겠습니다.